docs/developer-guide/ortools-integration.md

```diff
@@ -24,31 +25,36 @@ These are the solver names and enum, defined in the [OR-Tools API](https://githu
-| `cbc`                | CBC_MIXED_INTEGER_PROGRAMMING      | Free-to-use | [https://github.com/coin-or/Cbc](https://github.com/coin-or/Cbc)                                           |
-| `highs`              | HIGHS_MIXED_INTEGER_PROGRAMMING    | Free-to-use | [https://highs.dev](https://highs.dev)                                                                     |
-| `sirius` [^1]        | SIRIUS_MIXED_INTEGER_PROGRAMMING   | Free-to-use | [https://github.com/rte-france/sirius-solver](https://github.com/rte-france/sirius-solver)                 |
-| `gurobi`             | GUROBI_MIXED_INTEGER_PROGRAMMING   | Commercial  | [https://www.gurobi.com](https://www.gurobi.com)                                                           |
-| `cplex`              | CPLEX_MIXED_INTEGER_PROGRAMMING    | Commercial  | [https://www.ibm.com/fr-fr/analytics/cplex-optimizer](https://www.ibm.com/fr-fr/analytics/cplex-optimizer) |
-| `xpress`             | XPRESS_MIXED_INTEGER_PROGRAMMING   | Commercial  | [https://www.fico.com/en/products/fico-xpress-solver](https://www.fico.com/en/products/fico-xpress-solver) |
+
+| OR-Tools solver name | Enum                             | Usage       | Website                                                                                                    |
+|:---------------------|----------------------------------|-------------|------------------------------------------------------------------------------------------------------------|
+| `scip`               | SCIP_MIXED_INTEGER_PROGRAMMING   | Free-to-use | [https://www.scipopt.org](https://www.scipopt.org)                                                         |
+| `glpk`               | GLPK_MIXED_INTEGER_PROGRAMMING   | Free-to-use | [https://www.gnu.org/software/glpk](https://www.gnu.org/software/glpk)                                     |
+| `cbc`                | CBC_MIXED_INTEGER_PROGRAMMING    | Free-to-use | [https://github.com/coin-or/Cbc](https://github.com/coin-or/Cbc)                                           |
+| `highs`              | HIGHS_MIXED_INTEGER_PROGRAMMING  | Free-to-use | [https://highs.dev](https://highs.dev)                                                                     |
+| `sirius` [^1]        | SIRIUS_MIXED_INTEGER_PROGRAMMING | Free-to-use | [https://github.com/rte-france/sirius-solver](https://github.com/rte-france/sirius-solver)                 |
+| `gurobi`             | GUROBI_MIXED_INTEGER_PROGRAMMING | Commercial  | [https://www.gurobi.com](https://www.gurobi.com)                                                           |
+| `cplex`              | CPLEX_MIXED_INTEGER_PROGRAMMING  | Commercial  | [https://www.ibm.com/fr-fr/analytics/cplex-optimizer](https://www.ibm.com/fr-fr/analytics/cplex-optimizer) |
+| `xpress`             | XPRESS_MIXED_INTEGER_PROGRAMMING | Commercial  | [https://www.fico.com/en/products/fico-xpress-solver](https://www.fico.com/en/products/fico-xpress-solver) |
```

Comment:
I think only Sirius, XPRESS, SCIP and CBC are actually accessible from Antares Simulator